<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewTag</class>
 <widget class="QDialog" name="NewTag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Tag</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="mainGLayout">
     <item row="0" column="1" colspan="5">
      <widget class="QLabel" name="nameLabel">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="4">
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="colorButton">
       <property name="toolTip">
        <string>Choose color</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <spacer name="buttonHSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>135</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="4" colspan="2">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="mainVSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
